lab7/CMakeLists.txt

```
@@ -4,21 +4,22 @@ project(lab_7)
 add_compile_options(-std=c++14)
 
 find_package(catkin_simple REQUIRED)
+find_package(GTSAM REQUIRED)
 
 # Boost - same requirement as gtsam
 find_package(Boost 1.50 REQUIRED)
-include_directories(${Boost_INCLUDE_DIR})
+include_directories(${Boost_INCLUDE_DIR} ${GTSAM_INCLUDE_DIR})
```

```suggestion
include_directories(${Boost_INCLUDE_DIR})
```
No need to add include directories for gtsam here (the imported gtsam target from `find_package(GTSAM REQUIRED)` is an interface that brings them in when linking)